### ISSUE TITLE
Convert Doxygen-style comments to documentation comment

### ISSUE THREAD

**Issue by f0i** [dfinity/prettier-plugin-motoko#59]
Convert `//!` to `/// `, so the user doesn't have to think about that.

E.g. `//!` is used in [BitcoinWallet.mo](https://github.com/dfinity/examples/blob/master/motoko/basic_bitcoin/src/basic_bitcoin/src/BitcoinWallet.mo#L1)

**Comment by rvanasa** [comment]
Good idea--I'll look into combining this with #58 as an optional feature.
